mountpoint-s3/src/inode.rs

```diff
@@ -1502,8 +1509,8 @@ pub enum InodeError {
     IsDirectory(InodeErrorInfo),
     #[error("file already exists at inode {0}")]
     FileAlreadyExists(InodeErrorInfo),
-    #[error("inode {0} is not writable")]
-    InodeNotWritable(InodeErrorInfo),
+    #[error("inode {0} is not writable: {1:?}")]
+    InodeNotWritable(InodeErrorInfo, String),
```

Comment:
I think we should name the new variable so we don't have to guess what it is, something like
```suggestion
    #[error("inode {inode_error_info} is not writable because {error_reason:?}")]
    InodeNotWritable {
        inode_error_info: InodeErrorInfo,
        error_reason: String,
    },
```